src/librustdoc/json/mod.rs

```diff
@@ -217,10 +217,12 @@ impl<'tcx> FormatRenderer<'tcx> for JsonRenderer<'tcx> {
     fn after_krate(&mut self) -> Result<(), Error> {
         debug!("Done with crate");
 
+        /*
```

Comment:
This can just be removed, there's no point leaving it commented out. If we ever want to change the behaviour back, having this in the git history will be enough.